<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDeviceModelDlg_Eng</class>
 <widget class="QDialog" name="QDeviceModelDlg_Eng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>测试对象</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_btnSin">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnCoS">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnTan">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAsin">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAcos">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAtan">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnSind">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>sind</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnCosd">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>cosd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnTgd">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>tgd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAsind">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>asind</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAcosd">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>acosd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAtgd">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>atgd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnSqrt">
          <property name="minimumSize">
           <size>
            <width>58</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAbs">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>abs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnExp">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>exp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnAddition">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnSubtraction">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnMultiplication">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnDivision">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnLeftParenthesis">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnRightParenthesis">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QExprEdit" name="m_textEdit">
        <property name="maximumSize">
         <size>
          <width>16777</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>5</number>
       </property>
      <item>
       <widget class="QPushButton" name="m_btnEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>编辑</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnImport">
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnUpdate">
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnOk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QExprEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">../../../Module/UI/Controls/SettingCtrls/QExprEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
